lib/trufi_localizations.dart

```diff
@@ -946,6 +976,16 @@ abstract class TrufiLocalization {
 
   String instructionVehicleLightRail();
 
+  String instructionVehicleTrain();
```

Comment:
I believe these translations are not inside of Lokalise, it would be great to add them.